controllers/statusreport/statusreport_adapter.go

```diff
@@ -79,6 +79,16 @@ func (a *Adapter) EnsureSnapshotTestStatusReportedToGitHub() (controller.Operati
 		}
 	}
 
+	if gitops.IsSnapshotMarkedAsPassed(a.snapshot) || gitops.IsSnapshotMarkedAsFailed(a.snapshot) {
+		a.logger.Info("Status has been reported successfully, now in the process of removing the finalizer")
```

Comment:
we are inconsistent with logging here, for non-PAC we are not logging anything, for PAC PR, we are logging start and end of process.

Reply:
or maybe it's fine as it is